src/main/java/uk/co/ramp/covid/simulation/population/Transport.java

```diff
@@ -0,0 +1,41 @@
+package uk.co.ramp.covid.simulation.population;
+
+import uk.co.ramp.covid.simulation.output.DailyStats;
+import uk.co.ramp.covid.simulation.output.network.ContactsWriter;
+import uk.co.ramp.covid.simulation.parameters.PopulationParameters;
+import uk.co.ramp.covid.simulation.place.Place;
+import uk.co.ramp.covid.simulation.util.Time;
+
+/** Transport is a special form of Place where we allow infections to occur, but not to advance,
+ *  and no movements occur */
+public class Transport extends Place {
+
+    public Transport() {
+        transAdjustment = PopulationParameters.get().publicTransportParameters.transmissionConstant;
```

Comment:
Slightly concerned about this bit. Could we write an override for `Place.getTransConstant()`. In my view `Transport.getTransConstant()` should return 
`transConstant * transAdjustment / Population.populationSize`

The reason being that in its current formulation the Transport transmisison scales such that it is constant with the number of people that ar emoving at that time. In reality there will be more transmission when more people are using Transport. Does that make sense?